manifest-staging.yml

```diff
@@ -0,0 +1,21 @@
+---
+applications:
+- name: micropurchase-staging
+  memory: 1G
+  instances: 1
+  domain: 18f.gov
+  host: micropurchase-staging
+  buildpack: https://github.com/cloudfoundry/ruby-buildpack.git
```

Comment:
via Slack:

> i think that using `ruby_buildpack` will use the buildpack as installed on cloud.gov, as opposed to the cloud foundry version
> since the docs use both i’d say just leave it